frontend/src/api/WorkSheetAPI.js

```diff
@@ -1,11 +1,20 @@
 import api from './createApi'
+import moment from 'moment'
 
 function putWorkSheet(id, credentials) {
   return api.put(`/worksheets/${id}`, credentials)
 }
 
-function getWorkSheets() {
-  return api.get('/worksheets')
+function getWorkSheets(startDate, endDate) {
+  const minTime = startDate
+    ? moment(startDate).format('yyyy.MM.DD') + ' 00:00:00'
```

Comment:
Nagyon csúnya ez a string hozzáfűzés, hát már eleve használjuk a momentet meg a format-ját. RTFM, van startOf meg endOf, valamint a format is lehet ez: yyyy.MM.DD. HH:mm:ss

Reply:
Ha a format yyyy.MM.DD. HH:mm:ss, akkor az aktuális időpontot teszi hozzá. A másiknak utánanézek.